### DISCUSSION TITLE
Architecture testing for `declare(strict_types=1)` in files

### DISCUSSION THREAD

**Discussion by adevade** [pestphp/pest#813] [Ideas]
Is there a way to use the arch testing to make sure that files `declare(strict_types=1)`? 

If not, it might be a nice feature to add if possible? 😊

**Comment by nunomaduro** [comment]
@adevade I think that would be possible. cc @fabio-ivona

**Comment by fabio-ivona** [comment]
Will put it in my todos 👍😉

**Comment by adevade** [comment]
Was added in [v2.9](https://github.com/pestphp/pest/commit/be9056f9789a819870a01c686352957d89df264f) 😊
